python/assets/example_repo/pypi/shelf-reader/json/index.json

```diff
@@ -0,0 +1 @@
+{"info": {"author": "Austin Macdonald", "name": "shelf-reader", "summary": "Make sure your collections are in call number order."}, "releases": {"0.1": [{"path": "../../../source/s/shelf-reader/shelf-reader-0.1.tar.gz#md5=2dac570a33d88ca224be86759be59376", "md5_digest": "2dac570a33d88ca224be86759be59376", "packagetype": "sdist", "filename": "shelf-reader-0.1.tar.gz"}, {"path": "../../../source/s/shelf-reader/shelf_reader-0.1-py2-none-any.whl#md5=69b867d206f1ff984651aeef25fc54f9", "md5_digest": "69b867d206f1ff984651aeef25fc54f9", "packagetype": "bdist_wheel", "filename": "shelf_reader-0.1-py2-none-any.whl"}]}}
```

Comment:
@Ichimonji10, I changed this from index.html to index.json to make sure that the content type is correct. Whatever server is hosting the assets will need to forward a directory request to index.json as well as index.html.